dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -61,8 +61,11 @@ public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceC
     }
 
     public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, String configPath) throws Exception {
-        clearState();
```

Comment:
Is 1.4.0 being run for tests that use the non-confidential test data (line 56 seems unchanged above)